main.py

```diff
@@ -143,16 +154,44 @@ def postTeams(teams):
 
 
 def main():
```

Comment:
i think it would be easier to read if you'll split the logic to 4 separate functions. something like this:
```python
def main():
    match RUN_MODE:
        case "backup":
            backup()
        case "migrate":
            migrate()
        ...
```